tests/integration.rs

```diff
@@ -59,6 +59,10 @@ fn integration_test() {
         panic!("query stack during panic in the output");
     } else if stderr.contains("E0463") {
         panic!("error: E0463");
+    } else if stderr.contains("E0514") {
+        panic!("incompatible crate versions");
+    } else if stderr.contains("failed to run `rustc` to learn about target-specific information") {
+        panic!("couldn't find librustc_driver, consider setting `LD_LIBRARY_PATH`");
```

Comment:
These errors occured, while figuring out, that `RUSTUP_TOOLCHAIN=master` is required to execute the `integration` test binary successfully. I figured, that it can't harm to fail if these errors should happen when running integration tests.